invenio_rdm_records/resources/serializers/iiif/schema.py

```diff
@@ -135,9 +134,8 @@ def get_value(self, obj, *args, **kwargs):
         """Return the value for a given key from an object attribute."""
         return [
             f
-            for f in obj.files.entries
-            if splitext(f["key"])[1].replace(".", "").lower()
```

Comment:
does this change need be documented? Perhaps my understanding of this change is exaggerated, in that case then ignore :)